src/java.base/share/man/java.md

```diff
@@ -4004,6 +4004,110 @@ JVM will execute without loading any CDS archives. In addition, if
 you try to create a CDS archive with any of these 3 options specified,
 the JVM will report an error.
 
+## Ahead-of-Time Cache
+
+The JDK supports ahead-of-time (AOT) optimizations that can be performed before an
+application is executed. One example is Class Data Sharing (CDS), as described above,
+that parses classes ahead of time. AOT optimizations can improve the start-up and
+warm-up performance of Java applications.
+
+The Ahead-of-Time Cache (AOTCache) is a container introduced in JDK 24 for
+storing artifacts produced by AOT optimizations. The AOTCache currently contains
+Java classes and heap objects. The plans is to include other types of artifacts,
+such as execution profiles and compiled methods, in future JDK releases.
```

Comment:
s/plans/plan/ or reframe the sentence as:

> In future JDK releases, the AOT cache may contain additional artifacts, such as execution profiles and compiled methods.

Reply:
(Oh, and I still object to "AOTCache" as a term of art.  It should be "AOT cache".  I don't think anybody's manual of prose style allows CamelCase phrases.  Nobody wants to read a ManualOfProseStyle.)